### PR TITLE
All weighting branch changes

### PR DESCRIPTION
**Pull request by ARtheboss**
Signed-off-by: Advay Ratan <advayratan@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "saspes",
-  "version": "0.23.1",
+  "version": "0.24.0",
```
Comment (Suhas-13) [inline package.json:3]: this is supposed to be updated when a new release is actually pushed out, could you change this back?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
```
Comment (Suhas-13) [inline CHANGELOG.md]: again, this will be automatically populated once a new release is made, could you remove this change?

**Comment by Suhas-13** [comment]
@gary-kim would there be a way to have the new tests run on this without just creating a new PR?

**Comment by gary-kim** [comment]
> @gary-kim would there be a way to have the new tests run on this without just creating a new PR?

@Suhas-13 Any chance to the PR whether it be a new commit or a force push should run the new tests.

**Pull request closed without merging**